.github/workflows/ci.yml

```diff
@@ -135,7 +135,65 @@ jobs:
       - name: Run CI tests
         run: |
           cd tests/end_to_end_tests
-          
+
+          # Register database entries using dataregistry package
+          python create_test_entries.py
+
+          # Register more database entries using the dregs CLI
+          bash create_test_entries_cli.sh
+
+          # Run some test queries
+          pytest -v test_query.py
+
+  sqlite-end-to-end-tests:
+    runs-on: ubuntu-latest
+
+    env:
+      DREGS_CONFIG: "${{ github.workspace }}/config.txt"
+
+    # Our strategy lists the OS and Python versions we want to test on.
+    strategy:
+
+      # Don't quit all jobs if only one job fails.
+      fail-fast: false
+
+      matrix:
+        python-version: ["3.9"]
+
+    # Our CI steps for this job.
+    steps:
+      # Check out this repository code.
+      - name: Check out repository code
+        uses: actions/checkout@v3
+
+      # Install Python.
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      # Install our package.
+      - name: Install dataregistry
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install .
+          python -m pip install pytest
+
+      # Make a dummy config file.
+      - name: Create dummy config file
+        run: |
+          echo "sqlalchemy.url : sqlite:///${HOME}/registry.db" > $DREGS_CONFIG
```

Comment:
Just a note that this will fail when merged into main, as the name has been changed to DATAREG_CONFIG in the great DREGS purge.